src/components/shared/PinchZoomView.tsx

```diff
@@ -0,0 +1,100 @@
+import { ImageProps, SafeAreaView, TouchableOpacity, View } from 'react-native';
+import ImageViewer, { ImageViewerPropsDefine } from 'react-native-image-zoom-viewer';
+import React, { ReactElement } from 'react';
+import { Ionicons } from '@expo/vector-icons';
+import styled from 'styled-components/native';
+
+export interface ImageI extends HeaderProps {
+  url: string;
+  width?: number;
+  height?: number;
+  props?: ImageProps;
+}
+
+interface Props extends Omit<ImageViewerPropsDefine, 'imageUrls'> {
+  images: ImageI[];
+  onPressClose?: () => void;
+}
+
+const HeaderContainer = styled.SafeAreaView`
+  position: absolute;
+  width: 100%;
+  z-index: 1;  
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const Title = styled.Text`
+  color: white;
+  font-size: 20;
+`;
+
+const SubTitle = styled.Text`
+  color: white;
+  font-size: 16;
+`;
+
+interface HeaderProps {
+  title?: string;
+  subtitle?: string;
+  onPressClose?: () => void;
+}
+
+const Header = ({
+  title, subtitle, onPressClose,
+}: HeaderProps): React.ReactElement<HeaderProps, typeof SafeAreaView> => {
+  return (
+    <HeaderContainer>
+      <View style={{ marginLeft: 30 }}>
+        <Title>{title}</Title>
+        <SubTitle>{subtitle}</SubTitle>
+      </View>
+      <TouchableOpacity style={{ marginRight: 30 }} onPress={onPressClose}>
+        <Ionicons size={32} style={{ color: 'white' }} name="md-close" />
+      </TouchableOpacity>
+    </HeaderContainer>
+  );
+};
+
+const Nothing = (): ReactElement => {
+  return <></>;
+};
```

Comment:
I think this can be removed

Reply:
There is no way to hide indicator default if `renderIndicator` is not defined. 
Is it better to show indicator as `1/3` default? 

<img width="385" alt="image" src="https://user-images.githubusercontent.com/216363/69722052-863b0d80-1159-11ea-968b-41db10e4dfc5.png">